app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/activity/ExchangeFragment.java

```diff
@@ -0,0 +1,21 @@
+package ca.ualberta.CMPUT3012019T02.alexandria.activity;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import ca.ualberta.CMPUT3012019T02.alexandria.R;
+
+public class ExchangeFragment extends Fragment {
```

Comment:
Please rename to have ExchangeFragmentActivity

Reply:
But it is not an Activity, it is a Fragment. The fragment appears is in Main Activity. To call it an Activity is wrong I believe